charts/pega/charts/hazelcast/templates/_helpers.tpl

```diff
@@ -0,0 +1,38 @@
+{{- define "hazelcastName" -}} hazelcast {{- end -}}
+{{- define "hazelcastEnvironmentConfig" -}} hz-env-config {{- end -}}
+{{- define "hazelcastConfig" -}} hz-config {{- end -}}
+
+
+{{- define "isHazelcastEnabled" }}
+ {{- if .Values.enabled -}}
+  true
+ {{- else -}}
+  false
+ {{- end -}}
+{{- end }}
+
+{{- define "waitForHazelcast" -}}
+- name: wait-for-hazelcast
+  image: busybox:1.31.0
+  # Init container for waiting for hazelcast service to initialize.
+  args:
+  - sh
+  - -c
+  - >
+    set -e
+    counter=0;
+    while [ $(wget -q -O - "http://hazelcast-service.{{ .Release.Namespace }}:5701/hazelcast/health/cluster-size" /dev/null) -ne {{ .Values.hazelcast.replicas }} ] || [ $(wget -S "http://hazelcast-service.{{ .Release.Namespace }}:5701/hazelcast/health/cluster-safe" 2>&1 | grep "HTTP/" | awk '{print $2}') -ne 200 ]; do
```

Comment:
Service name can be taken from template like done at other places.  ` {{ template "hazelcastName" }}-service `

Reply:
I will check this

Reply:
Done.